packages/fsengagement/src/inboxblocks/CTABlock.tsx

```diff
@@ -70,6 +70,7 @@ export default class CTABlock extends Component<CTABlockProps> {
     cardPosition: PropTypes.number
   };
 
+  // // tslint:disable-next-line:cyclomatic-complexity
```

Comment:
tslint isn't picking this up, perhaps because of the `// //`

```
ERROR: /home/travis/build/brandingbrand/flagship/packages/fsengagement/src/inboxblocks/CTABlock.tsx:74:27 - The function has a cyclomatic complexity of 11 which is higher than the threshold of 10
```

Reply:
```suggestion
  // tslint:disable-next-line:cyclomatic-complexity
```